x-pack/plugins/maps/public/components/map/feature_tooltip.js

```diff
@@ -82,52 +89,49 @@ export class FeatureTooltip extends React.Component {
     }
 
     return (
-      <EuiFlexItem grow={false}>
-        <EuiButtonIcon
-          iconType="plusInCircle"
-          title={i18n.translate('xpack.maps.tooltip.filterOnPropertyTitle', {
-            defaultMessage: 'Filter on property'
-          })}
-          onClick={() => {
-            this.props.closeTooltip();
-            const filterAction = tooltipProperty.getFilterAction();
-            filterAction();
-          }}
-          aria-label={i18n.translate('xpack.maps.tooltip.filterOnPropertyAriaLabel', {
-            defaultMessage: 'Filter on property'
-          })}
-          className="mapFeatureTooltipFilterButton"
-        />
-      </EuiFlexItem>
+      <EuiButtonIcon
+        iconType="plusInCircle"
+        title={i18n.translate('xpack.maps.tooltip.filterOnPropertyTitle', {
+          defaultMessage: 'Filter on property'
+        })}
+        onClick={() => {
+          this.props.closeTooltip();
+          const filterAction = tooltipProperty.getFilterAction();
+          filterAction();
+        }}
+        aria-label={i18n.translate('xpack.maps.tooltip.filterOnPropertyAriaLabel', {
+          defaultMessage: 'Filter on property'
+        })}
+        className="mapFeatureTooltip__filterButton"
+      />
     );
   }
 
   _renderProperties(hasFilters) {
-    return this.state.properties.map((tooltipProperty, index) => {
+    return this.state.properties.map(tooltipProperty => {
       /*
        * Justification for dangerouslySetInnerHTML:
        * Property value contains value generated by Field formatter
        * Since these formatters produce raw HTML, this component needs to be able to render them as-is, relying
        * on the field formatter to only produce safe HTML.
        */
-      const htmlValue = (<span
-        // eslint-disable-next-line react/no-danger
-        dangerouslySetInnerHTML={{
-          __html: tooltipProperty.getHtmlDisplayValue()
-        }}
-      />);
-
-      return (
-        <EuiFlexGroup key={index}>
-          <EuiFlexItem grow={4}>
-            <strong>{tooltipProperty.getPropertyName()}</strong>
-          </EuiFlexItem>
-          <EuiFlexItem grow={6}>
-            {htmlValue}
-          </EuiFlexItem>
+      const htmlValue = (
+        <div>
+          <div
+            className="mapFeatureTooltip__lineDescription"
+            // eslint-disable-next-line react/no-danger
+            dangerouslySetInnerHTML={{
```

Comment:
Is there a reason to use this _dangerous_ prop instead of just explicitly  setting it as the child like: 
```jsx
<div className="mapFeatureTooltip__lineDescription">
  {tooltipProperty.getPropertyName()}
</div>
```

Reply:
The reason is that the value contains html. If you do not use `dangerouslySetInnerHTML` then the HTML will not be rendered as HTML but instead be placed in the as text. So instead of DOM, you would get an ugly string with HTML tags

Reply:
It would be great if the field formatters returned a react node but that is a much larger refactor

Reply:
Got it, thanks. It might be a good idea to throw a comment in the before the eslint-disable as to why it's necessary.